### PR TITLE
l10n screenshot tests: fix webview context menu, searchprovider test

### PR DESCRIPTION
**Pull request by garvankeeley**

### CODE REVIEW

**Comment by garvankeeley** [comment]
@isabelrios reminder that this PR will be needed if you try to run snapshot tests, the code was using Stefan's website, which is not longer up.

**Comment by isabelrios** [comment]
@garvankeeley I started to run the tests to check if the screenshots were taken in all languages and I realized that test `test06PanelsEmptyState` is only working in English, in other languages it fails due to the new way the panel is shown and how we have to dismiss it (using this [line](https://github.com/mozilla-mobile/firefox-ios/blob/master/XCUITests/FxScreenGraph.swift#L486) based on text)

What we can do, is instead of dismissing each panel, once we are in Bookmarks panel, without dismissing it, pressing on the other panels buttons which actually have IDs:
`LibraryPanels.Bookmarks`, `LibraryPanels.History`, `LibraryPanels.Downloads`, LibraryPanels.ReadingList` and if we are going to get screenshots from master we also need the 

Also, wondering if we want to screenshot also the new sync tab panel and the See all options as they are already in master (but not in this branch yet)

**Comment by garvankeeley** [comment]
Nice find, yeah lets do as you suggested.
Also, this line should be fixed to use an accessibility identifier: https://github.com/mozilla-mobile/firefox-ios/blob/master/XCUITests/FxScreenGraph.swift#L480

**Comment by isabelrios** [comment]
> Also, this line should be fixed to use an accessibility identifier: https://github.com/mozilla-mobile/firefox-ios/blob/master/XCUITests/FxScreenGraph.swift#L480

Yeah, I think I tried to add it when the panel view changed but did not find the way, I will look into that again

File: L10nSnapshotTests/L10nSnapshotTests.swift [hunk 1]
```diff
@@ -44,11 +44,17 @@ class L10nSnapshotTests: L10nBaseSnapshotTests {
...
-            snapshot("06PanelsEmptyState-\(i)-\(nodeName)")
-            i += 1
+        let libraryPanels = [
+            "LibraryPanels.History",
+            "LibraryPanels.ReadingList",
+            "LibraryPanels.Downloads",
+            "LibraryPanels.SyncedTabs"
+        ]
+        navigator.goto(LibraryPanel_Bookmarks)
+        snapshot("06PanelsEmptyState-LibraryPanels.Bookmarks")
```
Comment (isabelrios) [inline L10nSnapshotTests/L10nSnapshotTests.swift]: this should be: snapshot("06PanelsEmptyState-\(i)-\(libraryPanels[i])")

**Pull request merged**